libvisual-plugins/plugins/input/portaudio/Makefile.am

```diff
@@ -0,0 +1,9 @@
+## Process this file with automake to generate a Makefile.in
+
```

Comment:
Oops, we don't need `Makefile.am`.

Reply:
@kaixiong I think it slipped in while cherry-packing, sorry! Gone now like it never existed :smiley: